albumentations/augmentations/transforms.py

```diff
@@ -332,32 +336,70 @@ class ImageCompression(ImageOnlyTransform):
     """
 
     class InitSchema(BaseTransformInitSchema):
-        quality_lower: int = Field(default=99, description="Lower bound on the image quality", ge=1, le=100)
-        quality_upper: int = Field(default=100, description="Upper bound on the image quality", ge=1, le=100)
+        _size_len = 2
+        quality_range: Optional[Union[Tuple[int, int], List[int]]] = Field(
+            default=(99, 100),
+            description="lower and upper bound on the image quality as tuple (lower_bound, upper_bound)",
+        )
+        quality_lower: Optional[int] = Field(
+            default=None,
+            description="Lower bound on the image quality",
+            ge=1,
+            le=100,
+        )
+        quality_upper: Optional[int] = Field(
+            default=None,
+            description="Upper bound on the image quality",
+            ge=1,
+            le=100,
+        )
         compression_type: ImageCompressionType = Field(
             default=ImageCompressionType.JPEG,
             description="Image compression format",
         )
 
         @model_validator(mode="after")
-        def validate_quality(self) -> Self:
-            if self.quality_lower >= self.quality_upper:
-                msg = "quality_lower must be less than quality_upper"
-                raise ValueError(msg)
+        def validate_ranges(self) -> Self:
+            lower = 1
+            upper = 100
+
+            def validate_range(range: Tuple[int, int], name: str) -> None:
+                if len(range) != self._size_len:
+                    msg = f"'quality_range' should have lenght of {self._size_len} but lenght of {len(range)} was found"
+                    raise ValueError(msg)
+                for i, el in enumerate(range):
+                    if el < lower or el > upper:
+                        msg = f"element {i} in {name} has value {el} and not between {lower} and {upper}"
+                        raise ValueError
+                if range[0] > range[1]:
+                    msg = "quality lower bound must be less than quality upper bound"
+                    raise ValueError(msg)
+
+            if self.quality_lower or self.quality_upper:
```

Comment:
In pydantic 2.7 that we use, you add `deprectated` to the type definition in InitShema and if variable is used => deprecation warning will happen authomatically.

Example:

```python
min_holes: Optional[int] = Field(
            default=None,
            ge=0,
            description="Minimum number of regions to zero out.",
            deprecated="Use num_holes_range instead.",
        )
```
https://github.com/albumentations-team/albumentations/blob/main/albumentations/augmentations/dropout/coarse_dropout.py#L59

Reply:
`transform=A.ImageCompression()`

It is not optimal, but I think we can live with this till figure out better solution for warnings, or deprecate old interface (I think we can do it in fall)